app/src/main/java/org/android/go/sopt/data/datasource/remote/LogInDataSource.kt

```diff
@@ -0,0 +1,11 @@
+package org.android.go.sopt.data.datasource.remote
+
+import org.android.go.sopt.data.datasource.model.RequestLogInDto
+import org.android.go.sopt.data.datasource.model.ResponseLogInDto
+
+class LogInDataSource {
```

Comment:
logInService는 생성자로 넣어줍시당
```suggestion
class LogInDataSource(private val logInService: LoginService) { ....}
```